proposals/collection-expression-arguments.md

```diff
@@ -388,6 +388,20 @@ The key differences from the earlier algorithm are:
 * Candidate methods may have additional parameters *before* the `ReadOnlySpan<E>` parameter.
 * Multiple candidate methods are supported.
 
+### Applicable create method
+
+The definition of *applicable create method*, given a *collection arguments* list, is adapted from [*applicable function member*](https://github.com/dotnet/csharpstandard/blob/draft-v8/standard/expressions.md#12642-applicable-function-member):
+
+> **A method candidate is an *applicable create method* with respect to *collection arguments* `A`, possibly empty, when all of the following are true:**
+> * Each argument in `A` corresponds to a parameter **from the create method *excluding the last parameter***, at most one argument corresponds to each parameter, and any parameter to which no argument corresponds is an optional parameter.
+> * For each argument in `A`, the parameter-passing mode of the argument is identical to the parameter-passing mode of the corresponding parameter, and
+>   * for a value parameter ~~or a parameter array~~, an implicit conversion exists from the argument expression to the type of the corresponding parameter, or
```

Comment:
can we have a parameter-array (does that mean `params T[]...` btw, or something else?